blueocean-dashboard/src/main/less/run-pipeline.less

```diff
@@ -0,0 +1,35 @@
+.jdl-table {
+  .run-pipeline {
+    z-index: 2000;
+    font-size: 10px;
+    padding-left: 5px;
+    padding-top: 2px;
+    border-radius: 50%;
+    width: 20px;
+    height: 20px;
+    background-image: url('./icons/run.svg');
+    background-size: cover;
+  }
+
+  .run-pipeline:hover {
+    background-image: url('./icons/run-darkened.svg'); // TODO: nicer if it could be done via a data-uri transform
+  }
+
+  .run-pipeline-toast {
+    position: fixed;
+    left: 15px;
+    bottom: 15px;
+
+    // Need to set higher specificity rules for some of the
+    // Toast styles because they are being overridden by other
+    // styles in JDL (table styles).
+    .toast {
+      a.action {
+        color: @blueocean-blue;
+      }
+      .text {
+        white-space: normal;
+      }
```

Comment:
@cliffmeyers absolutely agree. The reason for adding it is in the comment before it. Basically ... the `Toast` component rendering in the DOM inside a `<table>` and the `<table>` has JDL styles that are overriding the `Toast` styles. Simple as that.

I had a quick look at the JDL styles in `toast.less` and can see that it does not specify these styles at all so is just inheriting. I will add a JDL JIRA to have these added + a comment on this code in this PR pointing to the JIRA.